src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -65,26 +78,26 @@
-    "death.attack.dragonMagic.player.item": "%1$s, %3$s kullanan %2$sden kaçmaya çalışırken ejderha nefesinde kavrultuldu.",
-    "death.attack.indirectDragonMagic": "%1$s, %2$s tarafından ejderha nefesinde kavrultuldu.",
-    "death.attack.indirectDragonMagic.item": "%1$s, %3$s kullanan %2$s tarafından ejderha nefesinde kavrultuldu.",
+    "death.attack.dragonMagic.player": "%1$s, %2$s'den kaçmaya çalışırken ejderha nefesinde kavruldu.",
+    "death.attack.dragonMagic.player.item": "%1$s, %3$s kullanan %2$s'den kaçmaya çalışırken ejderha nefesinde kavruldu.",
+    "death.attack.indirectDragonMagic": "%1$s, %2$s tarafından ejderha nefesinde kavruldu.",
+    "death.attack.indirectDragonMagic.item": "%1$s, %3$s kullanan %2$s tarafından ejderha nefesinde kavruldu.",
     "effect.toomanyorigins.end_fire": "End Ateşi",
 
-    "origin.legacytoomanyorigins.hare.description": "İnsanlara yakın derecede zekaya sahip ve yaptıkları oyuklarda yaşayan bir tavşan türü. Atalarından onlara hareketlilikleri miras kaldı.",
+    "origin.legacytoomanyorigins.hare.description": "İnsanlara yakın derecede zekaya sahip ve yaptıkları oyuklarda yaşayan bir tavşan türü. Atalarından onlara, hareketlilikleri miras kaldı.",
 
     "power.legacytoomanyorigins.bunny_hop.name": "Tavşan Zıplaması",
-    "power.legacytoomanyorigins.bunny_hop.description": "Havadayken momentumun gittikçe artar ve hareket hızına eklenir. Yere iniş yaptığında momentumun kısa bir süreden sonra sıfırlanır. Ayrıca özelliğinle momentum kazanabilirsin.",
+    "power.legacytoomanyorigins.bunny_hop.description": "Havadayken momentumun gittikçe artar ve hareket hızına eklenir. Topladığın momentum yerdeyken anlık olarak sende kalır. Ayrıca aktif gücün ile momentum kazanabilirsin.",
     "power.legacytoomanyorigins.moon_leap.name": "Uzun Şıçrama",
-    "power.legacytoomanyorigins.moon_leap.description": "Eğilirken, zıplama uzunluğun artar.",
+    "power.legacytoomanyorigins.moon_leap.description": "Eğilirken, zıplama gücün artar.",
     "power.legacytoomanyorigins.sugary_delicacy.name": "Şekerli Lezzet",
-    "power.legacytoomanyorigins.sugary_delicacy.description": "Havuç ve altın havuç yemek tavşan zıplaması momentumunu en fazla çeyreğine kadar doldurur.",
+    "power.legacytoomanyorigins.sugary_delicacy.description": "Havuç ve altın havuç yemek tavşan zıplaması momentumunu çeyreğine kadar doldurur.",
     "power.legacytoomanyorigins.photophobia.name": "Fotofobi",
     "power.legacytoomanyorigins.photophobia.description": "Yüksek ışıkta uyuyamazsın.",
     "power.legacytoomanyorigins.lightweight.name": "Zayıf",
-    "power.legacytoomanyorigins.lightweight.description": "İnsanlara göre 2 daha az kalbin var.",
+    "power.legacytoomanyorigins.lightweight.description": "İnsanlardan 2 daha az kalbin var.",
 
-    "subtitles.origin.hare.dash": "Hare şıçrar",
+    "subtitles.origin.hare.dash": "Hare şıçrıyor",
```

Comment:
idk why but they translate as "jumping" instead of "jumps" in the base translation, so I copied them for consistency, otherwise I think "jumps" makes more sense